addons/sys_intercom/fnc_vehicleInfoLine.sqf

```diff
@@ -0,0 +1,120 @@
+            if (_isBroadcasting && {_broadcastingUnit == acre_player}) then {
+                _textStatus = format ["<t font='PuristaBold' color='%1' size='0.6'>(R/B) </t>", _color];
+            } else {
+                if (_isVoiceActive) then {
+                    _textStatus = format ["<t font='PuristaBold' color='%1' size='0.6'>(R/T) </t>", _color];
+                } else {
+                    if (_unit getVariable [QGVAR(intercomPTT), false]) then {
+                        _textStatus = format ["<t font='PuristaBold' color='%1' size='0.6'>(R/P) </t>", _color];
+                    } else {
+                        _textStatus = format ["<t font='PuristaBold' color='%1' size='0.6'>(R/</t>", _color];
+                        _textStatus = _textStatus + format ["<t font='PuristaBold' color='#737373' size='0.6'>P/</t>"];
+                        _textStatus = _textStatus + format ["<t font='PuristaBold' color='%1' size='0.6'>) </t>", _color];
+                    };
+                };
+            };
+        };
+    };
+
+    private _voiceActivation = [_vehicle, _unit, _forEachIndex, INTERCOM_STATIONSTATUS_VOICEACTIVATION] call EFUNC(sys_intercom,getStationConfiguration);
+
+    _infoLine = _infoLine + format ["<t font='PuristaBold' color='%1' size='0.8'>%2 </t>", _color, _displayName] + _textStatus;
+} forEach _intercomNames;
+
+_infoLine = _infoLine + format ["<t font='PuristaBold' color='#ffffff' size='0.8'>| </t>"];
+
+{
+    if ([_x, _unit] call EFUNC(sys_rack,isRackAccessible) || [_x, _unit] call EFUNC(sys_rack,isRackHearable)) then {
+        private _rackClassName = _x;
+        private _config = ConfigFile >> "CfgVehicles" >> _rackClassName;
+        private _displayName = [_rackClassName, "getState", "shortName"] call EFUNC(sys_data,dataEvent);
+        private _mountedRadio = [_rackClassName] call EFUNC(sys_rack,getMountedRadio);
+        private _color = "#737373";
+        private _textStatus = "";
+        if (_mountedRadio in ACRE_ACCESSIBLE_RACK_RADIOS || {_mountedRadio in ACRE_HEARABLE_RACK_RADIOS}) then {
+            _color = "#ffffff";
+            _textStatus = "(R/T)";
+            if ([_x, _unit] call EFUNC(sys_rack,isRackHearable)) then {
+
+                private _connectionStatus = ["", _vehicle, _unit, _x] call FUNC(getRackRxTxCapabilities);
+                switch (_connectionStatus) do {
+                    case RACK_NO_MONITOR: {
+                        _textStatus = "";
+                    };
+                    case RACK_RX_ONLY: {
+                        _textStatus = "(R)";
+                    };
+                    case RACK_TX_ONLY: {
+                        _textStatus = "(T)";
+                    };
+                    case RACK_RX_AND_TX: {
+                        _textStatus = "(R/T)";
+                    };
+                };
+            };
+        };
+
+        _infoLine = _infoLine + format ["<t font='PuristaBold' color='%1' size='0.8'>%2 </t>", _color, _displayName];
```

Comment:
Again `_infoLine = format ["%1<t.....%1...%2...>", _infoLine, _color, _displayName];`
Rest of this file as well.